src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+    <p className="Person__partner">
+      {
+        person.isMarried
+          ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+          : 'I am not married'
+        }
```

Comment:
Try to avoid nested teranry.

```
{person.isMarried ? (
  <p className="Person__partner">
    {${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}}
  </p>
) :
  <p>I am not married</p>
}
```